compiler/ras/Debug.cpp

```diff
@@ -1092,7 +1092,7 @@ TR_Debug::nodePrintAllFlags(TR::Node *node, TR_PrettyPrinterString &output)
 #endif
    FLAG(chkStoreAlreadyEvaluated, "storeAlreadyEvaluated");
    FLAG(isCopyToNewVirtualRegister, "copyToNewVirtualRegister");
-   FLAG(chkNodeCreatedByPRE, "createdByPRE");
+   FLAG(isNodeCreatedByPRE, "createdByPRE");
```

Comment:
Looking at some of the methods associated with `Node` flags, it seems that `chkNodeCreatedByPRE` is odd in that it has a `TR_ASSERT`.  In other cases where methods associated with flags have `TR_ASSERT` calls, it seems to be the `OMR::Node::is<flag>` and `OMR::Node::set<flag>` methods that have calls to `TR_ASSERT` while `OMR::Node::chk<flag>` methods do not.

For instance, [`isReturnDummy`](https://github.com/eclipse/omr/blob/106c2f1519f88d1340efa7cbeffe77d578fc7615/compiler/il/OMRNode.cpp#L7504-L7509) and [`setReturnIsDummy`](https://github.com/eclipse/omr/blob/106c2f1519f88d1340efa7cbeffe77d578fc7615/compiler/il/OMRNode.cpp#L7511-L7516) have `TR_ASSERT` methods checking the opcode value, while [`chkReturnIsDummy`](https://github.com/eclipse/omr/blob/106c2f1519f88d1340efa7cbeffe77d578fc7615/compiler/il/OMRNode.cpp#L7518-L7522) does not.   It is `chkReturnIsDummy` that is used in [printing that `Node` flag](https://github.com/eclipse/omr/blob/106c2f1519f88d1340efa7cbeffe77d578fc7615/compiler/ras/Debug.cpp#L1085).

I believe it would be best to change the implementations of `chkNodeCreatedByPRE` and `isNodeCreatedByPRE` to follow suit.

Reply:
Thanks for the review - that makes sense. I've pushed an update with these changes.